### ISSUE TITLE
Add information on ephemeral storage for Astro

### ISSUE THREAD

**Issue by stuart23** [astronomer/docs#1014]
**Request type:**

- [ ] **New document:**: Add the `docs-content` label to this issue.
- [x] **Addition to an existing document**: Add the `enhancement` label to this issue.

**What is the purpose of the document/enhancement? Select all that apply:**

- [ ] Document a new feature for an upcoming release. Add the appropriate release version label to this issue.
- [ ] Information for an existing feature that isn't yet documented.
- [x] Additional considerations/ reference material for an existing document.

**Problem/feature that requires documentation:**

Customers keep running into issues where they exhaust the ephemeral storage on a node and their pod & task die. We would like to put some guidance around why this happens and different approaches to solve this.

There is a discussion [here](https://astronomer.slack.com/archives/C0128A6EBS8/p1659977901856139), but would still need some ideas from docs team about where this should live, and what format it should take.

**Are you the right person to review a draft of the document for technical accuracy? If not, who is?**

Yes

**Comment by jwitz** [comment]
That's a doozy of a thread you linked @stuart23 ! My general understanding is that there's no great way for customers to use ephemeral storage with their tasks because that's not how Airflow/ Astro was designed, and the KPO-based solutions are not quite ready to promote yet. 

If you know the specific error that appears when customers reach their storage limit, I think a good place to include at least "Don't do this"-level info is in https://docs.astronomer.io/astro/test-and-troubleshoot-locally#troubleshoot-common-issues

**Comment by stuart23** [comment]
Honestly, I think the starting point is just calling out the ephemeral storage restrictions that we have, and telling customers that if they try and use the local disk for any significant storage, then their tasks will fail. I don't think there is a good solution that we have ready for this right now, so I would be ok with saying that they can contact support for assistance on how to solve the problem for their use case.

Right now there is a lot of work going on for how we can make storage a supported feature, and I think we just want a stop gap that we can point customers to until we have a permanent solution.

**Comment by astrorfox** [comment]
I agree @stuart23. We can also direct users to the node types that have larger attached storage, right?

**Comment by kwcanuck** [comment]
It appears that this issue can be closed as the requested updates have been addressed. If this is still active, the issue can be reopened.
